pages/proposals/form/fields.js

```diff
@@ -0,0 +1,201 @@
+import validator from './validator';
+
+// *** IMPORTANT ***
+//
+// make sure all keys are all lowercase and contain no symbols or spaces
+// reference: https://www.npmjs.com/package/google-spreadsheet#row-based-api-limitations
+//
+// **********************************************
+
+let partOneFields = {
+  'firstname': {
+    type: `text`,
+    label: `First name`,
+    placeholder: `First name`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator()
+    ]
+  },
+  'surname': {
+    type: `text`,
+    label: `Surname`,
+    placeholder: `Surname`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator()
+    ]
+  },
+  'email': {
+    type: `text`,
+    label: `Email address`,
+    placeholder: `hello@example.com`,
+    labelClassname: `required`,
+    fieldClassname: `form-control`,
+    validator: [
+      validator.emptyValueValidator(),
+      validator.emailValidator()
+    ]
+  },
+  'organisation': {
+    type: `text`,
+    label: `Organisation `,
+    placeholder: `Organisation `,
+    fieldClassname: `form-control`,
+    validator: []
+  },
+  'twitterhandle': {
+    type: `text`,
+    label: `Twitter handle`,
+    placeholder: `Twitter handle`,
+    fieldClassname: `form-control`
+  },
+  'githubhandle': {
+    type: `text`,
+    label: `GitHub handle`,
+    placeholder: `Github handle`,
+    fieldClassname: `form-control`
+  },
+  'otherfacilitators': {
+    type: `text`,
+    label: `Additional facilitators for your session`,
+    placeholder: `Firstname Surname`,
+    fieldClassname: `form-control`,
+    multiplicity: 1,
+    addLabel: `+ Add another`
+  }
+};
+
+let partTwoFields = {
+  'space': {
+    type: `choiceGroup`,
+    label: `What space do feel your session will best contribute to?`,
+    options: [
+      `Web Literacy`,
+      `Decentralisation`,
+      `Open Innovation`,
+      `Digital Inclusion`,
+      `Privacy and Security`,
+      `Youth Zone`
+    ],
+    labelClassname: `required`,
+    fieldClassname: `form-control choice-group`,
+    validator: [
+      validator.emptyValueValidator()
+    ]
+  },
+  'alternatespace': {
```

Comment:
this field should be controlled by a checkbox that users can click, because right now there is no way to "unclick" this option. Once you pick one, you're stuck, you can't unpick it.